.github/workflows/gradle.yml

```diff
@@ -0,0 +1,51 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+
+name: SpringBoot CI with Gradle
+
+# Event trigger
+on:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    # Set running environment.
+    runs-on: ubuntu-latest
+    env:
+      working-directory: ./api
```

Comment:
Instead of using `env` and `${{ working-directory }}` for all steps, you can use default option. See https://github.com/orgs/community/discussions/25742#discussioncomment-3249014